front-end/src/app/shared/models/scha-transaction.model.ts

```diff
@@ -1,16 +1,9 @@
 import { plainToClass, Transform } from 'class-transformer';
-import { Transaction } from '../interfaces/transaction.interface';
+import { Transaction } from './transaction.model';
 import { LabelList } from '../utils/label.utils';
 import { BaseModel } from './base.model';
-import { Contact } from './contact.model';
-import { MemoText } from './memo-text.model';
 
-export class SchATransaction extends BaseModel implements Transaction {
-  id: string | undefined;
-
-  form_type: string | undefined;
-  filer_committee_id_number: string | undefined;
-  transaction_id: string | null = null; // This is a required field and must exist
+export class SchATransaction extends Transaction {
   back_reference_tran_id_number: string | undefined;
```

Comment:
If you moved `form_type` `filer_committee_id_number` `transaction_id` into the parent class Transaction, should we move these common fields as well? 
`back_reference_tran_id_number` `back_reference_sched_name` `entity_type` `election_code` `election_other_description` `conduit_name` `conduit_street1` `conduit_street2` `conduit_city` `conduit_state` `conduit_zip` `memo_code` `memo_text_description` `reference_to_si_or_sl_system_code_that_identifies_the_account`

Reply:
oh i bet you looked ahead at other schedules. nm